bbb-learning-dashboard/src/components/QuizzesTable.jsx

```
@@ -458,21 +462,21 @@ const QuizzesTable = (props) => {
       renderCell: (params) => {
         let type = 'default';
         const {
-          published, userAnswers, correctOption,
+          ended, userAnswers, correctOption,
         } = params?.row[params?.field];
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)